Codo-Baggins/refactor_tractor_fitlit
# Refactor - Scripts File Cleanup

# Pull Request Template

#### What’s this PR do?
This is a cleanup of `scripts.js` Specifically this branch: 
1. puts the User Data fetch in a function to be invoked on load.
1. reorders where functions are in relation to others in order to make the file more readable.
1. reorders where variable declarations and callbacks go within each function.
1. cleans up rogue semi colons


#### Where should the reviewer start?
Review all of `scripts.js`


#### How should this be manually tested?
1. Load the page a bunch of times.
1. Submit all of the forms.
Make sure nothing new breaks
